stable/dex-controller/templates/crd_connectors.yaml

```diff
@@ -0,0 +1,370 @@
+apiVersion: apiextensions.k8s.io/v1beta1
+kind: CustomResourceDefinition
+metadata:
+  creationTimestamp: null
+  name: connectors.dex.d2iq.io
```

Comment:
Nit: we are using Mesosphere in the API groups for some others CRDs. I raised this question in kommander channel but we might continue with that.

Reply:
Yes we have no `d2iq` api groups at all. Even when we do, it will be `d2iq.com` I think, but for now please stick to `mesopshere.io` domain.

Reply:
Done. I changed everything to `mesosphere.io`